src/lib/containers/FullTextSearch.tsx

```diff
@@ -60,10 +60,20 @@ export function FullTextSearch(props: FullTextSearchProps) {
           'org.sagebionetworks.repo.model.table.TextMatchesQueryFilter',
         searchExpression: searchText,
       }
+      // PORTALS-2093: does this additional filter already exist?
+      const found = additionalFilters.find(filter =>
+        filter.concreteType == textMatchesQueryFilter.concreteType &&
+        (filter as TextMatchesQueryFilter).searchExpression == textMatchesQueryFilter.searchExpression
+      )
+      if (found) {
+        return
```

Comment:
So basically, ignore the event entirely, and don't modify/clear the input? That makes sense to me. Maybe worth providing some feedback like above using `setCustomValidity` with something like `Search term ${term} is already present`

Reply:
Yes, ignore if already present.  Like the idea of adding additional feedback.   For now, since this is an edge case, going to merge in order to resolve the bug and can improve the UX later if prioritized.